projects/schematics/src/migrations/4_0/constructor-deprecations/data/currency.service.migration.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CURRENCY_SERVICE,
+  NGRX_STORE,
+  SITE_CONTEXT_CONFIG,
+  SPARTACUS_CORE,
+  SPARTACUS_STOREFRONTLIB,
+  STORE,
+  WINDOW_REF,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CURRENCY_SERVICE_MIGRATION: ConstructorDeprecation = {
```

Comment:
Nice you added schematics for all those breaking changes!
Do you plan also documenting them in 4_0.md migration doc in this PR?